operator/check_operator.go

```diff
@@ -84,11 +65,44 @@ func (c operatorCheck) Run(ctx context.Context) (certification.Results, error) {
 		return certification.Results{}, err
 	}
 
-	if err != nil {
-		return certification.Results{}, err
+	return eng.Results(ctx), nil
+}
+
+func (c *operatorCheck) resolve(ctx context.Context) error {
+	if !c.resolved {
```

Comment:
Can we switch this around to keep all of this code out of an indented block?

Reply:
fixed